main/process/dashboard.c

```diff
@@ -1876,6 +1878,86 @@ static void handle_display_theme(void)
     }
 }
 
+
+
+void generate_color_string(char* buffer, size_t size, const char* color_name, int color_value) {
+    snprintf(buffer, size, "%s{%d}", color_name, color_value);
```

Comment:
We usually 'const' int params where possible.
We usually JADE_ASSERT pointers and sizes/length params to functions.
We usually JADE_ASSERT the result of snprintf calls.

Reply:
Gotchya! will do. I will look for other places in the code to use JADE_ASSERT as well.